### PR TITLE
Test PullRequest no

### PR DESCRIPTION
**Pull request by nkhater**

### CODE REVIEW

File: jabref/Globals.java [hunk 1]
```diff
@@ -32,7 +32,9 @@
 public class Globals {
 	
 	// TODO Another line
-	// TODO Added a ToDo
+	//TODO
+	//TODO
+	//TODO
```
Comment (nkhater) [inline jabref/Globals.java]: My review

Comment (nkhater) [inline jabref/Globals.java]: My review comment

Comment (cqseTeamscale) [inline jabref/Globals.java]: Forgot Todo

File: jabref/Globals.java [hunk 2]
```diff
@@ -32,7 +32,9 @@
 public class Globals {
 	
 	// TODO Another line
-	// TODO Added a ToDo
+	//TODO
+	//TODO
```
Comment (nkhater) [inline jabref/Globals.java]: api review comment

File: jabref/JabRefGUI.java [hunk 3]
```diff
@@ -35,7 +35,7 @@
 
 public class JabRefGUI {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(JabRefGUI.class);
```
Comment (nkhater) [inline jabref/JabRefGUI.java]: :warning: Interface comment missing [(view in Teamscale)](https://c1df9117.ngrok.io/findings.html#details/github/?id=D9FA5A0DF86CF7EF3940ED7615A1400C&t=firstBranch:1560353915000)

File: jabref/JabRefGUI.java [hunk 4]
```diff
@@ -44,7 +44,7 @@
     private final List<ParserResult> failed = new ArrayList<>();
     private final List<ParserResult> toOpenTab = new ArrayList<>();
     private final DialogService dialogService;
-
+    // TODO
```
Comment (nkhater) [inline jabref/JabRefGUI.java]: :warning: TODO [(view in Teamscale)](https://c1df9117.ngrok.io/findings.html#details/github/?id=6D059CAA419631801CCC6F53C94D4B1D&t=firstBranch:1560353915000)

File: jabref/Globals.java [hunk 5]
```diff
@@ -32,7 +32,9 @@
 public class Globals {
 	
 	// TODO Another line
-	// TODO Added a ToDo
+	//TODO
```
Comment (nkhater) [inline jabref/Globals.java]: :warning: TODO [(view in Teamscale)](https://c1df9117.ngrok.io/findings.html#details/github/?id=C3D1B1BCAD10817AFE5C248386ECBE96&t=firstBranch:1560353915000)

File: jabref/Globals.java (hunk 2 above)
Comment (nkhater) [inline jabref/Globals.java]: :warning: TODO [(view in Teamscale)](https://c1df9117.ngrok.io/findings.html#details/github/?id=EE977BCB297102537B3B87109E369BDA&t=firstBranch:1560353915000)

**Pull request closed without merging**